pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go

```diff
@@ -1109,3 +1104,46 @@ func shouldBeIgnored(pod *v1.Pod) bool {
 		return time.Now().After(pod.DeletionTimestamp.Time.Add(time.Duration(*pod.DeletionGracePeriodSeconds) * time.Second))
 	}
 }
+
+func (gqm *GroupQuotaManager) deleteQuotaNoLock(quota *v1alpha1.ElasticQuota) error {
```

Comment:
Need to add some one-sided coverage to this function?

Reply:
done